lib/portManager.ts

```diff
@@ -1,15 +1,31 @@
 import axios from 'axios';
 
-import { PortManagerServer } from '../utils/PortManagerServer';
-import { detectPort } from '../utils/detectPort';
+import {
+  HEALTH_CHECK_PATH,
+  PortManagerServer,
+  SERVICE_HEALTHY,
+} from '../utils/PortManagerServer';
 import { PORT_MANAGER_SERVER_PORT } from '../constants/ports';
 import { RequestPortsData } from '../types/PortManager';
+import { detectPort } from '../utils/detectPort';
+import { logger } from './logger';
 
 export const BASE_URL = `http://localhost:${PORT_MANAGER_SERVER_PORT}`;
 
+export async function isPortManagerPortAvailable(): Promise<boolean> {
+  return (
+    (await detectPort(PORT_MANAGER_SERVER_PORT)) === PORT_MANAGER_SERVER_PORT
+  );
+}
+
```

Comment:
This is primarily for use with the `hs doctor` command since it is currently using `isPortManagerServerRunning` to check if the port is available and the logic of that is changing